src/jit/target.h

```diff
@@ -723,71 +722,134 @@ typedef unsigned short          regPairNoSmall; // arm: need 12 bits
 
   #define RBM_ALLINT              (RBM_INT_CALLEE_SAVED | RBM_INT_CALLEE_TRASH)
 #ifndef UNIX_AMD64_ABI
+#if !ETW_EBP_FRAMED
```

Comment:
In general, can you please add the "positive" case first, and the "negative" case in the "else" clause, to avoid double negative logic?